ImperatorToCK3/Data_Files/blankMod/output/common/decisions/10_religious_decisions.txt

```diff
@@ -0,0 +1,1102 @@
+						root.faith = {
+							faith_hostility_level_comparison = {
+								scope:current_county.var:false_convert <= scope:current_county.faith
+							}
+						}
+					}
+				}
+				faith = { has_doctrine = doctrine_pluralism_pluralistic }
+			}
+			any_vassal = {	#Otherwise, you can have a vassal who practices your faith in secret.
+				has_variable = false_convert
+				var:false_convert = root.faith
+			}
+			any_sub_realm_county = {	#... Or a county!
+				has_variable = false_convert
+				var:false_convert = root.faith
+			}
+		}
+	}
+
+	is_valid = {
+		is_adult = yes
+		NOR = {
+			has_trait = zealous
+			dread > low_dread
+			tyranny >= 1
+		}
+	}
+
+	cost = {
+		piety = major_piety_value
+	}
+
+	effect = {
+		trigger_event = false_conversion.0030
+		if = {
+			limit = {
+				faith = { has_doctrine = doctrine_pluralism_pluralistic }
+			}
+			custom_tooltip = encourage_confession_of_false_conversions_decision.desc.tt_pluralist
+			custom_tooltip = encourage_confession_of_false_conversions_decision.desc.tt_paranoia
+			custom_tooltip = encourage_confession_of_false_conversions_decision.desc.tt_pluralist_counties
+		}
+		else = {
+			custom_tooltip = encourage_confession_of_false_conversions_decision.desc.tt_coreligionist
+			custom_tooltip = revoke_false_conversion_decision.desc.tt_own_faith_counties
+		}
+	}
+
+	ai_potential = {
+		always = yes
+	}
+
+	ai_will_do = {
+		base = 25
+
+		modifier = {	#A tolerant liege, including yourself, makes this more likely.
+			add = 5
+			liege.faith = { has_doctrine = doctrine_pluralism_pluralistic }
+		}
+		modifier = {	#A tolerant top-lliege, including yourself, *also* makes this more likely.
+			add = 5
+			top_liege.faith = { has_doctrine = doctrine_pluralism_pluralistic }
+		}
+		modifier = {	#Planning for the future helps too.
+			add = 5
+			primary_heir.faith = { has_doctrine = doctrine_pluralism_pluralistic }
+		}
+		modifier = {	#If the faith isn't feeling militant, why should you?
+			add = 10
+			faith = { fervor <= medium_fervor_value }
+		}
+		modifier = {
+			add = 10
+			OR = {
+				has_trait = compassionate
+				has_trait = forgiving
+				has_trait = honest
+			}
+		}
+	}
+}
+
+go_on_pilgrimage_decision = {
+	picture = "gfx/interface/illustrations/decisions/decision_personal_religious.dds"
+	ai_check_interval = 60
+
+	desc = go_on_pilgrimage_decision_desc
+	selection_tooltip = go_on_pilgrimage_decision_tooltip
+
+	cooldown = { years = pilgrimage_cooldown_year_amount }
+
+	is_shown = {
+		is_landed = yes
+		NOR = {
+			# These Faiths historically shunned going on pilgrimages, saying they were a waste of time & money.
+			faith = faith:waldensian
+			faith = faith:paulician
+			faith = faith:lollard
+			faith = { # Removing pilgrimage option for pagans who never had such a thing attested
+				is_in_family = rf_pagan
+				NOT = {
+					faith = faith:hellenic_pagan
+					faith = faith:eleusinian_mysteries
+					faith = faith:mithraic_mysteries
+					faith = faith:arabic_pagan
+				}
+			}

```

Comment:
I'm still not convinced about the need for the addition, but if it's to be added it needs to be working.
Try
```suggestion
			AND = { # Removing pilgrimage option for pagans who never had such a thing attested
				religion = { is_in_family = rf_pagan }
				NOR = {
					faith = faith:hellenic_pagan
					faith = faith:eleusinian_mysteries
					faith = faith:mithraic_mysteries
					faith = faith:arabic_pagan
				}
			}

```